macros/draggableProof.pl

```diff
@@ -1,284 +1,404 @@
-# Done: show possible choices in TeX mode
-# To do: display student answers and correct answers in TeX mode properly.
-# To do: put jquery.nestable.js in a universal spot on every webwork server.
+################################################################
+=head1 NAME
+draggableProof.pl
 
-loadMacros("PGchoicemacros.pl");
+=head1 DESCRIPTION
```

Comment:
Delete the DESCRIPTION header.  It has no content.

Either that or add a description.

Reply:
A description has been filled in. Thank you!